mm/2s2h/BenGui/UIWidgets.cpp

```diff
@@ -1147,7 +1186,8 @@ void DrawFlagArray8Mask(const std::string& name, uint8_t& flags, Colors color) {
             }
         }
         if (ImGui::IsItemHovered()) {
-            std::string label = fmt::format("0x{:02X} ({})", bitMask, flagIndex);
+            std::string label = WrappedText(flagEntry.description, 60).c_str();
+            label += fmt::format("{}0x{:02X} ({})", label.size() ? "\n" : "", flagIndex, flagIndex);
```

Comment:
I'd like for the label to keep the bitMask, so we get something like `0x80 (7)` instead of `0x07 (7)`

```suggestion
            label += fmt::format("{}0x{:02X} ({})", label.size() ? "\n" : "", bitMask, flagIndex);
```